docs/debugger/hot-reload.md

```diff
@@ -43,21 +43,21 @@ We accomplish this by making it possible for you to edit your application's code
 
 The following table shows which application types support .NET Hot Reload with the debugger attached (F5) and without the debugger attached (Ctrl+F5), and whether .NET 6 is required for minimum support (that is, F5). .NET 6 is always required for Ctrl+F5 support. Also shown is the minimum version of Visual Studio that supports the feature.
 
-|Application type|.NET 6 required (F5)|F5|Ctrl+F5|
+|Application type|.NET 6 required|F5|Ctrl+F5|
 |-|-|-|-|
-|ASP.NET code behind|No|16.11|17.0|
+|ASP.NET code-behind|No|16.11|17.0|
 |ASP.NET Razor (Blazor Server and ASP.NET Core)|Yes|17.0|17.0|
-|ASP.NET Razor (Blazor WASM)|Yes|No|17.0|
+|ASP.NET Razor (Blazor WASM)|Yes|17.1|17.0|
 |WPF|No|16.11|17.0|
-|WinUI3|No|16.11|No|
+|WinUI3|No|16.11|--|
 |WinForms|No|16.11|17.0|
 |Console|No|16.11|17.0|
-|XAML .NET MAUI WinUI|Yes|17.1 Preview 1|No|
-|XAML .NET MAUI Android|Yes|17.1 Preview 1|No|
-|XAML .NET MAUI iOS|Yes|17.1 Preview 1|No|
-|XAML + Blazor .NET MAUI WinUI|Yes|17.1 Preview 1|No|
-|XAML + Blazor .NET MAUI Android|Yes|17.1 Preview 1|No|
-|XAML + Blazor .NET MAUI iOS|Yes|17.1 Preview 1|No|
+|.NET MAUI (WinUI 3)|Yes|17.1|--|
+|.NET MAUI (Android)|Yes|17.1|--|
+|.NET MAUI (iOS)|Yes|17.1|--|
+|.NET MAUI Blazor Hybrid (WinUI 3)|Yes|17.1|--|
+|.NET MAUI Blazor Hybrid (Android)|Yes|17.1|--|
+|.NET MAUI Blazor Hybrid (iOS)|Yes|17.1|--|
```

Comment:
The table looks OK to me. It's always good to do this kind of thing on the private repo, since we have build validation there, but we can see what happens.